crates/lexgen/tests/lua_5_1.rs

```diff
@@ -368,6 +368,7 @@ lexer! {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
I was thinking maybe `cfg(test)` would be better here, but somehow that doesn't fix the warning. I wonder why..

Not too important.

Reply:
Yes, I noticed this too. I think `cargo bench` sets `test` but I doesn't seem to set anything like `bench` or `benchmark`, that could be used with `#[cfg(not(bench))]` if it did. I gave up looking..